### ISSUE TITLE
Error message improvement

### ISSUE THREAD

**Issue by thekid** [xp-runners/reference#57]
``` sh
$ cat Test.class.php
<?php

class Test extends \util\cmd\Command {

  public function run() {
    $this->out->writeLine('Test');
  }
}

$ xp -m devel/xp/command/ Test
Uncaught exception: Exception lang.Error (Call to undefined static method lang.Object::main())
  at lang.Object::__callStatic((0x4)'main', array[1]) [line 365 of class-main.php]
```

This could be nicer

**Comment by thekid** [comment]
Solution for PHP 7.0+:

```diff
$ git diff
diff --git a/src/class-main.php b/src/class-main.php
index 0239cc0..defa472 100755
--- a/src/class-main.php
+++ b/src/class-main.php
@@ -96,4 +96,9 @@ try {
 } catch (\lang\SystemExit $e) {
   if ($message= $e->getMessage()) echo $message, "\n";
   exit($e->getCode());
+} catch (\Error $e) {
+  if (!is_callable([$class, 'main'])) {
+    throw new \Exception('Class `'.strtr($class, '\\', '.').'\' does not have a main() method');
+  }
+  throw $e;
 }
```

In lower PHP versions, we can only check beforehand.

**Comment by thekid** [comment]
In PHP 7.1, we might be able to use `Closure::fromCallable()` for this.

**Comment by thekid** [comment]
New behavior:

``` sh
$ xp Test
Uncaught exception: Exception (Class `Test' does not have a main() method)
  at <source> [line 365 of C:\tools\cygwin\home\friebe\bin\class-main.php]
  at <main>() [line 0 of Test.class.php]
```

**Comment by thekid** [comment]
:shipit: Released in [7.6.3](https://github.com/xp-runners/reference/releases/tag/v7.6.3)
